src/index.js

```diff
@@ -68,22 +67,11 @@ const ApifyClient = function (options = {}) {
         return (callOpts, callback) => {
             const mergedOpts = Object.assign({}, instanceOpts, callOpts);
 
+            // Remove traling forward slash from baseUrl.
+            if (mergedOpts.baseUrl.substr(-1) === '/') mergedOpts.baseUrl = mergedOpts.baseUrl.slice(0, -1);
```

Comment:
ok just beware that `mergedOpts.baseUrl` might be null so this will throw an error, maybe the below check should be earlier and of form `if (! mergedOpts.baseUrl)`